tests/quantization/test_quantized_ops.py

```diff
@@ -1193,81 +1249,66 @@ def test_quantized_prelu(n_bits, input_range, input_shape, slope, is_signed, che
     )
 
 
-@pytest.mark.parametrize(
-    "params",
-    [
+@pytest.fixture(scope="module")
+def random_test_data():
+    """Generate data for comparisons operators."""
+    numpy.random.seed(42)  # Fixing the seed for reproducibility
```

Comment:
shouldn't we make it random like other tests : `random_state = numpy.random.randint(0, 2**15)` ? If we fix seeds like this, we'll never encounter some possible corner cases that would need to be fixed !

Reply:
Yes right I should have removed that. I used a fix seed for debugging. I should probably use pytest args instead. I will remove.

Reply:
Done